### PR TITLE
Added gallery API

### PR DESCRIPTION
**Pull request by ujjwal123123**
I am not sure about schema, so tell if it needs to be changed.

### CODE REVIEW

File: models/landingGallery.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const landingGallerySchema = new mongoose.Schema({
+  title: String,
+  description: String,
+  coverImage: String,
+  date: Date,
+  url: String,
+});
+
+module.exports = new mongoose.model('landingImage', landingGallerySchema);
```
Comment (i-rebel-aj) [inline models/landingGallery.js:11]: Kindly explain your model, either discuss in a group or provide comments. Ideally adding endpoints and models should each have a different PR.

File: controllers/API.js [hunk 2]
```diff
@@ -23,4 +24,6 @@ router.use(require('./API/POST/xclubGalleryPage.js'));
 router.use(require('./API/GET/xclubGalleryPage.js'));
 router.use(require('./API/DELETE/xclubGalleryPage.js'));
 
+router.use(require('./API/GET/gallery.js'));
+
 module.exports = router;
```
Comment (i-rebel-aj) [inline controllers/API.js:29]: You can commit this file.

File: controllers/API/GET/gallery.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+// });
+
+// newGallery.save((err, gallery) => {
+//   if (err) {
+//     console.log(err);
+//   } else {
+//     console.log(gallery);
+//   }
+// });
+
```
[the remark this answers is not in the document]
Comment (ujjwal123123) [reply controllers/API/GET/gallery.js:20]: Noted.

File: controllers/API/GET/gallery.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+console.log(landingGallery.find());
+
+router.get('/api/techboard/gallery', (req, res) => {
+  console.log('Working on techboard/gallery');
+  landingGallery
+    .find({}, (err, gallery) => {
+      if (err) {
+        res.statusCode = 500;
+        console.error('Error occured while fetching gallery');
+        res.send('Error occured while fetching gallery');
```
[the remark this answers is not in the document]
Comment (ujjwal123123) [reply controllers/API/GET/gallery.js:30]: Is it okay if I replace the code with --

``` javascript
res.statusCode = 500;
console.error('Error occured while fetching gallery');
res.json({ error: err.message });
```

File: controllers/API/GET/gallery.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      } else {
+        console.log(gallery);
+        console.log('sending techboard/gallery');
```
[the remark this answers is not in the document]
Comment (ujjwal123123) [reply controllers/API/GET/gallery.js:33]: All of the `console.log` statements are for debugging and none of them is really needed. Should I remove them?

File: models/landingGallery.js (hunk 1 above)
Comment (ujjwal123123) [reply models/landingGallery.js:11]: Noted. After discussing this in the group, I will send two different PRs. Maybe, this PR should be closed?

File: controllers/API/GET/gallery.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (i-rebel-aj) [reply controllers/API/GET/gallery.js:33]: Yes

File: controllers/API/GET/gallery.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (i-rebel-aj) [reply controllers/API/GET/gallery.js:30]: You can do it in one line..... res.json({ success: false, msg: 'Something' }); only a matter of personal preference.

**Pull request closed without merging**